<launch>
<!--    pure nmpc controller-->
    <group ns="fhnp">
        <node pkg="oop_qd_onbd" type="nmpc_node.py" name="nmpc_node" output="screen"/>
<!--        mavros_node-->
    </group>

</launch>
